### PR TITLE
337727: TopicSubscription Name updated 

### PR DESCRIPTION
**Pull request by ganeshg2491**
# **What this PR does / why we need it**:
Changes related to change for topicsubscription naming convention in ASO helm library
Related PR:
https://github.com/DEFRA/adp-aso-helm-library/pull/27

Relates to ADO Work Item [AB#337727](https://dev.azure.com/defragovuk/93c65135-d16b-49d6-a191-4a5313532779/_workitems/edit/337727)

# **Special notes for your reviewer**
*Any specific actions or notes on review?*

# Testing
Tested CI build for 'ffc-demo-payment-service'
https://dev.azure.com/defragovuk/DEFRA-FFC/_build/results?buildId=500227&view=results

# Checklist (please delete before completing or setting auto-complete)
- [x] Story Work items associated (not Tasks)
- [x] Successful testing run(s) link provided
- [x] Title pattern should be `{work item number}: {title}`
- [x] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests

# **How does this PR make you feel**:
gif

### CODE REVIEW

**Review by DaRoszko — APPROVED** [review]

**Review by jeevankuduvaravindran — APPROVED** [review]

File: appConfig/appConfig.yaml [hunk 1]
```diff
@@ -5,11 +5,11 @@
 - key: PAYMENT_TOPIC_ADDRESS
   value: "ffc-demo-payment"
 - key: PAYMENT_SUBSCRIPTION_ADDRESS
-  value: "payment"
+  value: "ffc-demo-payment"
 - key: SCHEDULE_TOPIC_ADDRESS
   value: "ffc-demo-schedule" 
 - key: SCHEDULE_SUBSCRIPTION_ADDRESS
-  value: "payment"  
+  value: "ffc-demo-payment"
```
Comment (rajeshkaremane) [inline appConfig/appConfig.yaml:12]: Should be : ffc-demo-schedule-payment

File: appConfig/appConfig.yaml [hunk 2]
```diff
@@ -5,11 +5,11 @@
 - key: PAYMENT_TOPIC_ADDRESS
   value: "ffc-demo-payment"
 - key: PAYMENT_SUBSCRIPTION_ADDRESS
-  value: "payment"
+  value: "ffc-demo-payment"
```
Comment (rajeshkaremane) [inline appConfig/appConfig.yaml:8]: ffc-demo-payment-payment

Comment (ganeshg2491) [reply appConfig/appConfig.yaml:8]: @rajeshkaremane : No, it is **namespace-topicsubscriptionname** . so above is right.

File: appConfig/appConfig.yaml (hunk 1 above)
Comment (ganeshg2491) [reply appConfig/appConfig.yaml:12]: @rajeshkaremane : No, format is **namespace-topicsubscriptionname** . so above is right.

Comment (rajeshkaremane) [reply appConfig/appConfig.yaml:12]: Should be a separate ticket then

File: appConfig/appConfig.yaml (hunk 2 above)
Comment (rajeshkaremane) [reply appConfig/appConfig.yaml:8]: Should be a separate ticket then

**Review by kbabigumira — APPROVED** [review]

**Pull request merged**